IssueReporter/Assets/en.lproj/IssueReporter-Localizable.strings

```diff
@@ -1,16 +1,8 @@
-/* 
-  IssueReporter-Localizable.strings
-  Pods
-
-  Created by Hakon Hanesand on 7/27/15.
-
-*/
-
-"Camera" = "Camera";
-"Photo library" = "Photo library";
+"Camera" = "Capture image";
+"Photo library" = "Library";
 "Cancel" = "Cancel";
-"Pick image" = "Pick image";
+"Pick image" = "Select image";
 
-"What went wrong?" = "Hva gikk galt?";
-"New Issue" = "Ny Issue";
-"Title" = "Tittel";
\ No newline at end of file
+"What went wrong?" = "What went wrong?";
```

Comment:
English translation for norwegian addition so we keep similar translations :
"Inkluder gjerne din epost, slik at vi kan ta kontakt ved behov for mer informasjon"
"If you include your email, we can contact you if we need more information"